### PR TITLE
Updated docker config for qid-batch-runner

### PR DESCRIPTION
**Pull request by ryangrundy7**
# Motivation and Context

I've updated qid-batch-runner to be able to put files in different sftp locations so the unaddressed feature needs to be updated with new config.

# What has changed

- Updated config to have new environment variables

# How to test?

- Run ATs with [qid-batch-runner branch](https://github.com/ONSdigital/census-rm-qid-batch-runner/pull/38)

# Links

[Trello](https://trello.com/c/Ij81s0mo/728-rmc-319b-generate-ce-spg-initial-contact-letter-unaddressed-print-files-ew-8)

### CODE REVIEW

**Comment by ryangrundy7** [comment]
[RMC-319B Generate CE/SPG Initial contact letter Unaddressed Print Files (E+W) (8)](https://trello.com/c/Ij81s0mo/728-rmc-319b-generate-ce-spg-initial-contact-letter-unaddressed-print-files-ew-8)

File: acceptance_tests/features/steps/unaddressed.py [hunk 1]
```diff
@@ -125,10 +125,12 @@ def validate_unaddressed_print_file(context):
              '--env', 'DB_HOST=postgres',
              '--env', 'DB_PORT=5432',
              '--env', 'OUR_PUBLIC_KEY_PATH=dummy_keys/our_dummy_public.asc',
-             '--env', 'OTHER_PUBLIC_KEY_PATH=dummy_keys/supplier_dummy_public.asc',
+             '--env', 'QM_PUBLIC_KEY_PATH=dummy_keys/supplier_QM_dummy_public.asc',
+             '--env', 'PPO_PUBLIC_KEY_PATH=dummy_keys/dummy_ppo_supplier_public_key.asc',
```
Comment (NickGrantX) [inline acceptance_tests/features/steps/unaddressed.py]: would be nice if these keys were consistently named

Comment (ryangrundy7) [reply acceptance_tests/features/steps/unaddressed.py]: Yeah I'll do that! Was meaning to do it

**Review by AdamHawtin — APPROVED** [review]

**Review by NickGrantX — APPROVED** [review]
LGTM 👍

**Pull request merged**